magicctapipe/scripts/lst1_magic/lst1_magic_mc_dl0_to_dl1.py

```diff
@@ -466,17 +509,26 @@ def main():
         "-c",
         dest="config_file",
         type=str,
-        default="./config.yaml",
+        default="./config_step1.yaml",
         help="Path to a configuration file",
     )
+    
+    parser.add_argument(
+        "--focal_length_choice",
+        "-f",                                 
+        dest="focal_length_choice",
+        type=str,
+        default="effective",
+        help='Standard is "effective"',
```

Comment:
Help can be a bit more verbose :) even if the name of the option is quite explicative, adding a few more words is ok e.g. "Choice of focal length, either effective or nominal. The standard is "effective"". Actually I am not sure the other choice is nominal, maybe equivalent? for that reason it is also better to add the `choices` parameter

```python
choices=["effective", "nominal"]
```
so that the parser will also tell when wrong strings are passed.

Reply:
In ctapipe 0.12 SimtelEventSource the other one is nominal
![image](https://github.com/cta-observatory/magic-cta-pipe/assets/121040436/a38b8be2-7244-4c70-88c3-7e04c4c36bb6)